MoviesTVSentiments/app/src/main/java/com/google/moviestvsentiments/service/web/Resource.java

```diff
@@ -0,0 +1,99 @@
+package com.google.moviestvsentiments.service.web;
+
+/**
+ * A container for a value, its loading status and an error message.
+ * @param <T> The type of value stored.
+ */
+public class Resource<T> {
```

Comment:
Wondering if it is possible to use autovalue for this?

Reply:
Also does it need equals? https://drive.google.com/file/d/1mW8CaRfZmeTu_lnf3EdBStCntVpqcnSB/view?usp=sharing